docs/cody/clients/model-configuration.mdx

```diff
@@ -0,0 +1,520 @@
+
+## Concepts
+
+The LLM models available for use from a Sourcegraph Enterprise instance are the union of "Sourcegraph-supplied models" and any custom models providers that you explicitlly add to your Sourcegraph instance's site configuration. For most administrators, just relying on Sourcegraph-supplied models will ensure that you are using quality models without needing to worry about the worry about the specifics.
+
+### Sourcegraph-supplied Models
+
+The Sourcegraph-supplied models are those that are available from [Cody Gateway](/cody/core-concepts/cody-gateway), and your site configuration controls which of those models can be used.
+
+If you wish to not use _any_ Sourcegraph-supplied models, and instead _only_ rely on those you have explicitly defined in your site configuration, you can set the `"sourcegraph"` field to `null`.
+
+There are three top-level settings for configuring Sourcegraph-supplied LLM models:
+
+| Field                     | Description                                                                              |
+| -----------               | ---------------------------------------------------------------------------------------- |
+| `endpoint` (optional)     | The URL for connecting to Cody Gateway, defaults to the production instance.             | 
+| `accessToken` (optional)  | The access token used to connect to Cody Gateway, defaulting to the current license key. |
+| `modelFilters` (optional) | Filters for which models to include from Cody Gateway.                                   |
+
+**Model Filters**
+
+The `"modelFilters"` section is how you restrict which Cody Gateway models are made available to your Sourcegraph Enterprise instance's users.
+
+The first field is the `"statusFilter"`. Each LLM model is given a label by Sourcegraph as per its release, such as "stable", beta", or "experimental". By default, all models available on
+Cody Gateway are exposed. Using the category filter ensures that only models with a particular category are made available to your users.
+
+The `"allow"` and `"deny"` fields, are arrays of model references for what models should or should not be included. These values accept wild cards.
+
+
+The following examples illusrate how to use all these settings in conjunction:
+
+```json
+"modelConfiguration": {
+  "sourcegraph": {
+    "modelFilters": {
+      // Only allow "beta" and "stable" models.
+      // Not "experimental" or "deprecated".
+      "statusFilter": ["beta", "stable"],
+
+      // Allow any models provided by Anthropic or OpenAI.
+      "allow": [
+        "anthropic::*",
+        "openai::*"
+      ],
+
+      // Do not include any models with the Model ID containing "turbo",
+      // or any from AcmeCo.
+      "deny": [
+        "*turbo*",
+        "acmeco::*"
```

Comment:
Should we link this to the Supported Models Matrix, with the configuration names ?